### DISCUSSION TITLE
MenuItemModel route to external URL

### DISCUSSION THREAD

**Discussion by elray** [kevinpapst/TablerBundle#192] [General]
Hi Kevin

I want to add an external link to the users details menu, is this possible? 

Following is giving me a 500 error.

`$event->addLink(new MenuItemModel('my_menu_id', 'my_menu_label', 'https://www.domainname.com/', [], 'fas fa-company-icon'));`

regards
Eliot

**Comment by creiner** [comment]
Not directly, its always passed to the path function.
https://github.com/kevinpapst/TablerBundle/blob/c4fd884b8b5dbc12449f0b3130de790fee4d3566/templates/includes/navbar_user.html.twig#L26

You could create a dummy action which only does the redirect to your page and link to this action.

**Reply by elray** [reply]
Thanks for the pointer @creiner thats just what I was looking for
